overlay/hooks/entrypoint-pre.d/10_generate_config.sh

```diff
@@ -188,6 +189,9 @@ if ! [ -z "${PASSTHRU_IPS}" ]; then
   done                                                                                
 fi                                                                                      
 
+# Add line for custom domains
+echo "\$INCLUDE $CUSTOM_ZONE" >> ${RPZ_ZONE}
```

Comment:
Do we need a blank custom.db in the overlay file to prevent errors/warnings related to this not existing in the include?

Reply:
Or perhaps an `if ! file exists then touch file` type affair

Reply:
This was added with 562164b0656b6d128ad22df83de119d47cc474e7.

We don't really need this, as it just ignores the include if the file didn't exist. I couldn't find that documented anywhere other than my testing, so I added it just to be safe.